src/main/java/org/broadinstitute/hellbender/tools/BwaMemIndexImageCreator.java

```diff
@@ -19,25 +23,51 @@
 )
 public final class BwaMemIndexImageCreator extends CommandLineProgram {
 
+    public static final String USE_EXISTING_INDEX_FULL_NAME = "useExistingIndexFilesIfAvailable";
+    public static final String USE_EXISTING_INDEX_SHORT_NAME = "useIndexFiles";
+
     @Argument(fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
-              shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
-              doc = "Input reference fasta file. The five bwa index files are assumed living in the same directory with the same prefix.")
+            shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
+            doc = "Input reference fasta file. The five bwa index files are assumed living in the same directory with the same prefix.")
```

Comment:
I think that it is much better to move the "The five bwa index files are assumed living in the same directory with the same prefix." to the `USE_EXISTING_INDEX_FULL_NAME` arguments, and include here what @mwalker174 suggested.